osf/models/institution.py

```diff
@@ -164,21 +164,21 @@ def logo_path(self):
         try:
             return self.asset_files.get(name='logo').file.url
         except InstitutionAssetFile.DoesNotExist:
-            return None
+            return '/static/img/institutions/shields/placeholder-shield.png'
 
     @property
     def logo_path_rounded_corners(self):
         try:
             return self.asset_files.get(name='logo_rounded_corners').file.url
         except InstitutionAssetFile.DoesNotExist:
-            return None
+            return '/static/img/institutions/shields/placeholder-shield.png'
```

Comment:
should this point to the rounded corner one?

e.g. for staging3, https://staging3.osf.io/static/img/institutions/shields-rounded-corners/placeholder-shield-rounded-corners.png